src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -257,48 +191,17 @@ namespace winrt::TerminalApp::implementation
         // this as a MTA, before the app is Create()'d
         WINRT_ASSERT(_loadedInitialSettings);
 
-        _root->DialogPresenter(*this);
-
         // In UWP mode, we cannot handle taking over the title bar for tabs,
         // so this setting is overridden to false no matter what the preference is.
         if (_isUwp)
         {
             _settings.GlobalSettings().ShowTabsInTitlebar(false);
         }
 
-        // Pay attention, that even if some command line arguments were parsed (like launch mode),
-        // we will not use the startup actions from settings.
-        // While this simplifies the logic, we might want to reconsider this behavior in the future.
-        if (!_hasCommandLineArguments && _hasSettingsStartupActions)
+        // These used to be in `TerminalPage::Initialized`, so that they started

```

Comment:
nit: markers for where code used to be will fall out of date pretty much immediately when we rewrite `TerminalPage::Initialized`. Is this a helpful comment?

I guess I don't know what it's trying to tell me

Reply:
Y'know, that's more of a not to self, ain't it. I guess it's a TODO?